scripts/provision/camera_monitor_profile.yml

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright (c) 2018
+# Dell
+# Mainflux
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# File taken from https://docs.edgexfoundry.org/Ch-WalkthroughDeviceProfile.html
+
+name: "camera monitor profile"
+manufacturer: "Dell"
+model: "Cam12345"
+labels: 
+    - "camera"
+description: "Human and canine camera monitor profile"
+commands: 
+  - 
+    name: People
+    get: 
+        path: "/api/v1/devices/{deviceId}/peoplecount"
+        responses:
+          - 
+            code: "200"
+            description: "Number of people on camera"
+            expectedValues: ["humancount"]
+          -
+            code: "503"
+            description: "service unavailable"
+            expectedValues: ["cameraerror"]
+  - 
+    name: Dogs
+    get: 
+        path: "/api/v1/devices/{deviceId}/dogcount"
+        responses:
+          - 
+            code: "200"
+            description: "Number of dogs on camera"
+            expectedValues: ["caninecount"]
+          -
+            code: "503"
+            description: "service unavailable"
+            expectedValues: ["cameraerror"]
+  - 
+    name: ScanDepth
+    get:
+        path: "/api/v1/devices/{deviceId}/scandepth"
+        responses:
+          -
+            code: "200"
+            description: "Get the scan depth"
+            expectedValues: ["depth"]
+          -
+            code: "503"
+            description: "service unavailable"
+            expectedValues: ["cameraerror"]    
+    put:
+        path: "/api/v1/devices/{deviceId}/scandepth"
+        parameterNames: ["depth"]
+        responses:
+          - 
+            code: "204"
+            description: "Set the scan depth."
+            expectedValues: []
+          -
+            code: "503"
+            description: "service unavailable"
+            expectedValues: ["cameraerror"]
+  - 
+    name: SnapshotDuration
+    get:
+        path: "/api/v1/devices/{deviceId}/snapshotduration"
+        responses:
+          -
+            code: "200"
+            description: "Get the snaphot duration"
+            expectedValues: ["duration"]
+          -
+            code: "503"
+            description: "service unavailable"
+            expectedValues: []    
+    put:
+        path: "/api/v1/devices/{deviceId}/SnapshotDuration"
+        parameterNames: ["duration"]
+        responses:
+          - 
+            code: "204"
+            description: "Set the snapshot duration."
+            expectedValues: []
+          -
+            code: "503"
+            description: "service unavailable"
+            expectedValues: ["cameraerror"]
+  - 
+    name: Counts
+    get:
+        path: "/api/v1/devices/{deviceId}/counts"
+        responses:
+          -
+            code: "200"
+            description: "Get the people and human counts"
+            expectedValues: ["count","count"]
+          -
+            code: "503"
+            description: "service unavailable"
+            expectedValues: []

```

Comment:
As defined in [Device Services functional requeriments](https://wiki.edgexfoundry.org/download/attachments/7602423/edgex-cali-delhi-device-requirements-v8.pdf?version=1&modificationDate=1533761857000&api=v2): _A DeviceProfile must at minimum define a single DeviceObject_, and this is not the case. Should be fine if this example is according to specifications.